hcov/bio_methods.py

```diff
@@ -96,7 +98,7 @@ def sortAndExtractUnmapped(script: TextIOWrapper, options: OptionsDict, referenc
     processUnmapped = options["processUnmapped"]
     alignOnly = options["alignOnly"]
 
-    sortAlignedAndMappedData(script, options, reference)
+    sortedBamFile = sortAlignedAndMappedData(script, options, reference)
```

Comment:
almost feels like there should be a function that can generate the aligned / sorted bam names and those get used instead of burying the construction (my fault) and then trying to pass names around

Reply:
good call, added a new `buildBamFilePath` function for that